test/e2e/lib/validators.js

```diff
@@ -388,10 +388,21 @@ const createValidateArrivalOrDeparture = (type, cfg) => {
 		}
 
 		val.line(val, dep.line, name + '.line')
-		if (type === 'departure') {
-			const n = name + '.direction'
-			a.strictEqual(typeof dep.direction, 'string', n + ' must be a string')
-			a.ok(dep.direction, n + ' must not be empty')
+
+        if (type === 'departure') {
```

Comment:
Can you convert these lines to tabs, which are used in the `hafas-client` code base. This will also make the diff more "focused".